File: facebook/app/getPostId.py
import requests

# Replace with your actual access token
access_token = '695503689341265|H1qKJCaoyrhiT7HBDOKXoigzK2k'

api_url = f'https://graph.instagram.com/me/media?fields=id,caption&access_token={access_token}'

response = requests.get(api_url)

if response.status_code == 200:
    posts = response.json().get('data', [])
    for post in posts:
        print('Post ID:', post['id'])
        print('Caption:', post.get('caption', 'No Caption'))
        print('---')
else:
    print('Failed to fetch posts:', response.status_code, response.text)

File: facebook/app/getPostId.js
const accessToken = '695503689341265|H1qKJCaoyrhiT7HBDOKXoigzK2k'; // Replace with your actual access token

fetch(`https://graph.instagram.com/me/media?fields=id,media_type,media_url,caption,permalink&access_token=${accessToken}`)
  .then(response => {
    if (!response.ok) {
      return response.json().then(err => {
        throw new Error(`HTTP error! Status: ${response.status}. Error: ${JSON.stringify(err)}`);
      });
    }
    return response.json();
  })
  .then(data => {
    console.log('Post Data:', data);
    // Process and display the data
  })
  .catch(error => {
    console.error('Error fetching Instagram media:', error);
  });

